Source/WeatherContent/WeatherView.swift

```diff
@@ -0,0 +1,76 @@
+//  Copyright © 2016 Gavan Chan. All rights reserved.
+
+import UIKit
+
+final class WeatherView: UIView {
```

Comment:
Something / Some things that we could probably factor out of the view controller and *into* this view are the information label(s) that are currently being managed by the view controller include the weather information labels (since they are technically part of the `WeatherView`. In this vein, the Weather view controller could then manage simply whether or not to display this view based on whether it has any content to show (and as such, whether or not to show the activity indicator).

#### note: 
This may lead to further abstraction down the road to have a `ContentTransitioning` protocol, for example, which describes types which have either are "showing" content or "not-showing" content based on whether they have content or not, extracting the way such types display visual feedback depending on whether or not they have content (which in our case is similar between the two use cases--which simply show an activity indicator in the center of their respective views).